src/mapgen.cpp

```diff
@@ -491,7 +491,7 @@ static mapgen_factory oter_mapgen;
 
 std::map<nested_mapgen_id, nested_mapgen> nested_mapgens;
 std::map<update_mapgen_id, update_mapgen> update_mapgens;
-
+static std::unordered_map<std::string, std::string> queued_points;
```

Comment:
```suggestion
static std::unordered_map<std::string, tripoint_abs_ms> queued_points;
```
And stringify only in `set_queued_points()`. Then you won't have to convert twice in `map::rotate()`